README.rst

```diff
@@ -109,12 +109,18 @@ these using Anaconda::
 
     $ conda install numpy geos
 
-On Linux / OSX::
+On OSX::
 
     $ export GEOS_INCLUDE_PATH=$CONDA_PREFIX/Library/include
     $ export GEOS_LIBRARY_PATH=$CONDA_PREFIX/Library/lib
     $ pip install pygeos
 
+On Linux::
+
+    $ export GEOS_INCLUDE_PATH=$CONDA_PREFIX/include
+    $ export GEOS_LIBRARY_PATH=$CONDA_PREFIX/lib
+    $ pip install pygeos
```

Comment:
Not purely related to your edit since it is already in there right now (and your fix is certainly correct), but: is exporting those env variables actually needed? 
When installing with conda, it should provide the `geos-config` command in the environment, and then the `get_geos_paths` in `setup.py` should automatically find geos in the conda env (at least I am not setting any evn variables, and it is working for me).

Reply:
I'm not sure it's required. I have adjusted the paths as I've seen they were wrong. Due to your feedback, trying in a virtual environment, like you, I was able to make successfully the install e.g:

```bash
    conda create -n pygeodemo python=3.6
    conda activate pygeodemo
    pip install pygeos
```

Reply:
I remember when I tried installing with conda, `geos-config` was not available through conda. But now I try again I see that it is actually there.

So I am in favor of removing the setting of the two variables. I have no idea how this would work on a Mac. Anybody got experience with that?

Reply:
A difference might be that you need to activate your environment first or restart the console before it is available.

No experience with Mac.

I would remove setting in the default instructions, but maybe we should still mention that those can be set in case GEOS is not found automatically.